src/control/lib/daos/api/util.go

```diff
@@ -105,10 +114,16 @@ func ranklistFromGo(rs *ranklist.RankSet) *C.d_rank_list_t {
 	return rl
 }
 
-func mustLogCtx(parent context.Context, log logging.Logger) context.Context {
+func mustLogCtx(parent context.Context, t *testing.T) context.Context {
```

Comment:
seems strange to include a testing package primitive inside `api/util.go` shouldn't it be inside a test specific file?

Reply:
... Yes. Will move in a follow-up.

Reply:
https://github.com/daos-stack/daos/pull/15982